packages/module/src/LogSnippet/LogSnippet.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { CodeBlock, CodeBlockCode, Flex, FlexItem, } from '@patternfly/react-core';
+
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  logSnippet: {
+    borderLeft: '2px solid var(--pf-v5-global--danger-color--100)',
```

Comment:
The red border should be probably optional or at least overridable with custom class. We may also need a different variant than danger

Reply:
@nicolethoen please, what do you think about the border vs PF habits? Do we want to have it visible by default with an option to hide and select color variant? Or maybe do not show it and if OpenShift wants it, they can pass a custom className? Thank you

Reply:
There's a few ways it could be done. 
An additional prop could be used - either a flag like `isBorderLeft`, or an enum like `borderLeftVariant='red|<other options>`.
we could even expose a `customBorderLeftColor` which could override this style if we want red to be the default?

Reply:
I would probably vote for borderLeftVariant with colors and "hidden" option. Thoughts?

Reply:
Yeah I'm good with that. I'll get to it.

Reply:
@nicolethoen is there a subset of acceptable colors? I just went ahead and added `'red' | 'green' | 'blue' | 'cyan' | 'gold' | 'orange' | 'purple'`